app/controllers/remit_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RemitController < ApplicationController
+  def new
+    @event = Event.find_by(id: params[:id])
```

Comment:
Ah I see. It's because of the routes. If a manager gets remitted from his events, then the routes are wrong. I'm not too sure how it works any more so perhaps we can meet today?